pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -72,6 +72,33 @@ const (
 	uninstallHCOErrorMsg  = "The uninstall request failed on dependent components, please check their logs."
 )
 
+var conditions = []conditionsv1.Condition{
+	{
+		Type:    conditionsv1.ConditionAvailable,
+		Status:  corev1.ConditionTrue,
+		Reason:  reconcileCompletedMessage,
+		Message: reconcileCompleted,
+	},
+	{
+		Type:    conditionsv1.ConditionProgressing,
+		Status:  corev1.ConditionFalse,
+		Reason:  reconcileCompletedMessage,
+		Message: reconcileCompleted,
+	},
+	// {
+	// 	Type:    conditionsv1.ConditionDegraded,
```

Comment:
So - don't we monitor components's degraded condition at all?

Reply:
fixed